packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -766,15 +782,15 @@ class _DragTargetState<T extends Object> extends State<DragTarget<T>> {
     });
     if (avatar.data != null)  {
       widget.onAccept?.call(avatar.data! as T);
-      widget.onAcceptWithDetails?.call(DragTargetDetails<T>(data: avatar.data! as T, offset: avatar._lastOffset!));
+      widget.onAcceptWithDetails?.call(DragTargetDetails<T>(data: avatar.data! as T, offset: avatar._lastOffset));
```

Comment:
Maybe your IDE is smarter than the analyzer, it seems like it needs the `!` to pass.